sound/soc/sof/hw-spi.c

```diff
@@ -233,8 +232,8 @@ static int spi_sof_probe(struct snd_sof_dev *sdev)
 	struct platform_device *pdev =
 		container_of(sdev->dev, struct platform_device, dev);
 	struct snd_sof_pdata *sof_pdata = sdev->pdata;
-	struct sof_spi_dev *sof_spi =
-		(struct sof_spi_dev *)sof_pdata->hw_pdata;
+	struct sof_spi_dev *sof_spi = sof_pdata->hw_pdata;
+
```

Comment:
why an empty line now?

Reply:
@lyakh its just a style preference. some people prefer an empty line before comments and I'm one of them.
But maybe it makes sense to remove it from variable declarations. I'll delete it.